### PR TITLE
Fixes #23782 - restore login disabling

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: app/views/settings/category/_email.html.erb [hunk 1]
```diff
@@ -1,3 +1,5 @@
-<%= link_to_function(_("Test email"), "tfm.users.testMail(this, '#{test_mail_user_url(User.current)}')",
-                     :title => _("Send a test message to the current user's email address to confirm the configuration is working."), :id => "test_mail_button", :class =>"btn btn-success") + hidden_spinner('', :id => 'test_indicator').html_safe %>
+<% if Setting[:login] %>
+  <%= link_to_function(_("Test email"), "tfm.users.testMail(this, '#{test_mail_user_url(User.current)}')",
+                       :title => _("Send a test message to the current user's email address to confirm the configuration is working."), :id => "test_mail_button", :class =>"btn btn-success") + hidden_spinner('', :id => 'test_indicator').html_safe %>
 <%= content_tag(:br) %>
```
Comment (adamruzicka) [inline app/views/settings/category/_email.html.erb]: Shouldn't this be indented as well?

**Review by adamruzicka — COMMENTED** [review]
Works for me, left one nitpick inline

File: app/views/settings/category/_email.html.erb (hunk 1 above)
Comment (ares) [reply app/views/settings/category/_email.html.erb]: done :-)

**Review by adamruzicka — APPROVED** [review]
LGTM

**Review by tbrisker — APPROVED** [review]
Thanks @ares !

**Comment by tbrisker** [comment]
Please also CP to 1.17 and 1.18

**Pull request merged**